### PR TITLE
Fixing all peer review issues 

### LINKED ISSUE #80: Peer Feedback - inputs of `stat_summary` and `generate_virtual_pop`
_For the purposes of consistency, I would consider either making the inputs for `generate_virtual_pop` and `stat_summary` that are numpy objects both either numpy objects or strings (if you choose to make both strings, my comment about adding `import numpy as np` to Usage is irrelevant)_

### LINKED ISSUE #81: Peer Feedback - `stat_summary` documentation
documentation for  function `[stat_summary](https://samplingsimulatorpy.readthedocs.io/en/latest/source/samplingsimulatorpy.html#module-samplingsimulatorpy.stat_summary)` is wrongly formatted.

### LINKED ISSUE #83: Peer Feedback - `stat_summary` tests
- `test_stat_summary`: It is a good practice to have a short introduction to the test function like the other test functions in your package. However, users might be a little bit  confused about what the `TypeError` error is, which argument the test is referring to. Therefore, a more explicit expression is expected.

### PR DESCRIPTION
**Pull request by tguo9**

### CODE REVIEW

File: samplingsimulatorpy/generate_virtual_pop.py [hunk 1]
```diff
@@ -34,7 +33,8 @@ def generate_virtual_pop(size, population_name, distribution_func, *para):
 
     Examples
     --------
-    >>> from samplingsimulatorpy import generate_virtual_pop
+    >>> from samplingsimulatorpy.generate_virtual_pop
+                                                import generate_virtual_pop
```
Comment (hwilliams10) [inline samplingsimulatorpy/generate_virtual_pop.py]: Is this indentation correct?

**Review by hwilliams10 — APPROVED** [review]
One minor suggestion. 
Also, it might be helpful to add an example of the type of distribution functions that can be used for the `generate_virtual_pop` documentation (ex. np.random.normal).

File: samplingsimulatorpy/generate_virtual_pop.py (hunk 1 above)
Comment (tguo9) [reply samplingsimulatorpy/generate_virtual_pop.py]: I'm afraid I have to do this, otherwise flake8 will complain about it

Comment (tguo9) [reply samplingsimulatorpy/generate_virtual_pop.py]: I will make them on the same vertical line. Thanks!

Comment (tguo9) [reply samplingsimulatorpy/generate_virtual_pop.py]: Done!

**Pull request merged**